modules/hostdb.go

```diff
@@ -7,7 +7,7 @@ import (
 
 const (
 	// Denotes a host announcement in the Arbitrary Data section.
-	HostAnnouncementPrefix = "HostAnnouncement"
+	PrefixHostAnnouncement = "HostAnnouncement\n"
```

Comment:
hmm. You don't think it might be JSON encoded?

This stuff is easy enough to change later, I guess I can remove it for now.